### PR TITLE
Add trackEventWithOptionsAndMeasurements method

### PR DESCRIPTION
**Pull request by brianfoody**
Tracking events with name doesn't really allow us to provide information around these events. I have added a new method called trackEventWithOptionsAndMeasurements which plugs into the hockey app method for providing options and measurements with each event.

### CODE REVIEW

File: android/src/main/java/com/slowpath/hockeyapp/RNHockeyAppModule.java [hunk 1]
```diff
@@ -186,6 +187,15 @@ public void trackEvent(String eventName) {
     }
   }
 
+   @ReactMethod
+  public void trackEventWithOptionsAndMeasurements(String eventName, Map<String,String> properties, Map<String,Double> measurements) {
+    if(_initialized)
+    {
+      log(eventName);
+       MetricsManager.trackEvent(eventName, properties, measurements));
```
Comment (iamladi) [inline android/src/main/java/com/slowpath/hockeyapp/RNHockeyAppModule.java:195]: You're missing once space on 194 or there's one unnecessary on this line.

File: RNHockeyApp/RNHockeyApp.m [hunk 2]
```diff
@@ -131,6 +131,18 @@ @implementation RNHockeyApp
     }
 }
 
+RCT_EXPORT_METHOD(trackEventWithOptionsAndMeasurements:(NSString *)eventName Options:(NSDictionary *)options Measurements:(NSDictionary *)measurements)
```
Comment (iamladi) [inline RNHockeyApp/RNHockeyApp.m:134]: Why not modify the `trackEvent` directly instead of creating a new method?

**Comment by dsternlicht** [comment]
Hey @martincik ! 

What's the status of this pull request? Are you going to merge it soon?

**Comment by iamladi** [comment]
I tried to reach @brianfoody with a feedback, but he's not replying and I want to know the answers before I merge it. I don't want to merge this just because it's contribution.

@brianfoody Can you get back to me with the feedback.

BTW: Thanks so much for the contribution. Very much appreciated.

**Pull request merged**